src/envoy/BUILD

```diff
@@ -29,6 +29,7 @@ envoy_cc_binary(
         "//src/envoy/http/jwt_auth:http_filter_factory",
         "//src/envoy/http/mixer:filter_lib",
         "//src/envoy/tcp/mixer:filter_lib",
+        "//src/envoy/tcp/tcp_cluster_rewrite:tcp_cluster_rewrite_lib",
```

Comment:
config_lib? tcp_cluster_rewrite_lib doesn't have the static registration.

Reply:
Addressed in #2021.